### PR TITLE
Bug fix: change descriptions for some success messages

### PR DESCRIPTION
**Pull request by LenaNguyen**
Adding a participant successfully:
Updated to include participant name.

Successfully registering for a class:
This was grammatically incorrect.

### Checklist

-   [x] My PR name is descriptive and in imperative tense
-   [x] I have run the linter
-   [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by MatoPlus** [comment]
I think we originally had this in our design:
image

@LenaNguyen Do you mind tackling this? Otherwise, lmk and I'll make a ticket to take a look on a later date!
I think it's just passing participant name states around.

**Comment by LenaNguyen** [comment]
> I think we originally had this in our design: image
> 
> @LenaNguyen Do you mind tackling this? Otherwise, lmk and I'll make a ticket to take a look on a later date! I think it's just passing participant name states around.

updated! 
image

File: src/components/parent-form/ParticipantCreatedPage.tsx [hunk 1]
```diff
@@ -112,9 +114,15 @@ export const ParticipantCreatedPage: React.FC<ParticipantCreatedPageProps> = ({
...
                         <Text maxW={400} textAlign="center" pt={3} pb={9}>
-                            {successful === "success"
-                                ? "Participant has been successfully added as a participant in your account"
-                                : "There was an error adding the participant. Please contact us"}
+                            {successful === "success" ? (
+                                <div>
+                                    <strong>{name}</strong> has been
+                                    successfully added as a participant in your
+                                    account!
+                                </div>
```
Comment (MatoPlus) [inline src/components/parent-form/ParticipantCreatedPage.tsx]: We want to minimize the amount of non-chakra components, especially div.
```suggestion
                                <>
                                    <strong>{name}</strong> has been
                                    successfully added as a participant in your
                                    account!
                                </>
```

**Review by MatoPlus — CHANGES_REQUESTED** [review]
One last little thing and I think it's good to merge!

**Review by MatoPlus — APPROVED** [review]
LGTM!

**Pull request merged**